<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReporteEspecifico</class>
 <widget class="QDialog" name="ReporteEspecifico">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reporte de paciente</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reporte:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Tipo de ejercicio:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Fecha:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Tipo de onda:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTipoOnda">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxxxxxxxxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Dracion: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVideo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Visualizar Ondas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReportes">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>520</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Visualizar Reporte</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEmviarReporte">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>520</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Imprimir Reporte</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Terapeuta: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Tutor: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbDuracion">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxxxxxxxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTerapeuta">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxxxxxxxxxxxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTutor">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sin_Tutor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Paciente:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPaciente">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxxxxxxxxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbComentarios">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>551</width>
     <height>121</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 255, 185);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbTipo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxxxxxxxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbFecha">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxxxxxxxxxxxxxxxxxxxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbIdentificador">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Numero</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Meta alcanzada:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMetaAlcanza">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>430</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>xx</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbEjercicio_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>de</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMetaPropuesta">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>xx</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>251</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/newPrefix/EjerElevar.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAbrirGrafico">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>520</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Visualizar Grafico</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>600</width>
     <height>641</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(210, 220, 220), stop:1 rgba(100, 130, 190));

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 198, 223);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>19</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 75 19pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Datos de un Reporte especifico</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/newPrefix/reporte.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>601</width>
     <height>8</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>8</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(214, 225, 229), stop:1 rgba(109, 130, 198));
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Edad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbFecha_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxxxxxxxxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTipo_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de ejercicio:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 8pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Nombre de ejercicio:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>440</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Tipo de Grafico</string>
   </property>
   <widget class="QRadioButton" name="rbGrafLIneal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lineal</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbGrafBarras">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Barras</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <zorder>label_8</zorder>
  <zorder>label</zorder>
  <zorder>label1</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>lbTipoOnda</zorder>
  <zorder>label_4</zorder>
  <zorder>btnVideo</zorder>
  <zorder>btnReportes</zorder>
  <zorder>btnEmviarReporte</zorder>
  <zorder>label_6</zorder>
  <zorder>label_9</zorder>
  <zorder>lbDuracion</zorder>
  <zorder>lbTerapeuta</zorder>
  <zorder>lbTutor</zorder>
  <zorder>label_10</zorder>
  <zorder>lbPaciente</zorder>
  <zorder>lbComentarios</zorder>
  <zorder>lbTipo</zorder>
  <zorder>lbFecha</zorder>
  <zorder>lbIdentificador</zorder>
  <zorder>label_5</zorder>
  <zorder>lbMetaAlcanza</zorder>
  <zorder>lbEjercicio_3</zorder>
  <zorder>lbMetaPropuesta</zorder>
  <zorder>label_7</zorder>
  <zorder>btnAbrirGrafico</zorder>
  <zorder>frame_3</zorder>
  <zorder>frame_4</zorder>
  <zorder>label_12</zorder>
  <zorder>lbFecha_2</zorder>
  <zorder>lbTipo_2</zorder>
  <zorder>label1_2</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources>
  <include location="images/image.qrc"/>
 </resources>
 <connections/>
</ui>
